package.json

```diff
@@ -28,40 +28,39 @@
     "test": "ember try:each"
   },
   "dependencies": {
-    "blob-polyfill": "~1.0.20150320",
-    "broccoli-funnel": "^1.2.0",
-    "broccoli-merge-trees": "^1.2.4",
+    "blob-polyfill": "^2.0.20171115",
+    "broccoli-funnel": "^2.0.1",
+    "broccoli-merge-trees": "^2.0.0",
     "broccoli-stew": "^1.5.0",
-    "chalk": "^1.1.1",
-    "ember-cli-babel": "^6.3.0",
+    "chalk": "^2.3.0",
+    "ember-cli-babel": "^6.6.0",
     "ember-cli-string-utils": "^1.1.0",
-    "ember-cli-version-checker": "^1.3.1",
-    "ember-getowner-polyfill": "^1.2.3"
+    "ember-cli-version-checker": "^2.1.0"
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.5.0",
     "ember-ajax": "^3.0.0",
-    "ember-cli": "^2.14.0",
-    "ember-cli-dependency-checker": "^1.4.0",
-    "ember-cli-eslint": "^3.1.0",
-    "ember-cli-github-pages": "0.1.2",
+    "ember-cli": "~2.16.2",
```

Comment:
Can you update to `2.17.0` which got released 2 days ago?

Reply:
Done, cli and other dependencies are up to date again.